_dashboards/discover/visualizing-ad.md

```diff
@@ -0,0 +1,158 @@
+
+<insert UI>
+
+## Exploring alerting monitor details
+
+Once you have created or associated alerting monitors, you can verify the monitor is generating the alerts and explore the alert details by following these steps:
+
+1. From the dashboard, view the <**name**> visualization. Alerts are indicated with a red triangle. 
+1. Hover over a triangle to view the alert count details. To investigate the alert details, select a triangle to activate the pop-up window containing the monitor details. Alternatively, select the ellipsis icon in the visualization pane and choose **View events** to view those details.
+1. Select the ellipsis icon, then **Alerting** > **Associated monitors**.
+1. From the Associated monitors window, view the monitor list and then choose an alerting monitor, for example, <**CPU Usage Across World**>. Monitor details such as history, alerts, and associated visualizations, are shown from within the visualization pane.
+1. _Optional_: From the Associated monitors window, unlink a monitor from the visualization by selecting the link icon under **Actions**. This unlinks the monitor from the visualization only; it does not delete the monitor itself.
+1. _Optional_: From the Associated monitors window, edit the monitor's metrics, for example, the threshold for CPU usage. 
+
+Monitor details are viewable from the dashboard and visualization pane, as shown in the following image:
+
+<insert UI>
+
+## Creating anomaly detectors
+
+Anomaly detectors identify unusual patterns or outliers in a dataset. Once  you have defined your problem and prepared your data, you can load it into OpenSearch for indexing. In the following steps, you will continue using the sample dataset that you used in the preceding steps.
+
+To create an anomaly detector using the Discover application and OpenSearch sample data and then visualize the anomalies on a dashboard, follow these steps: 
+
+1. From the OpenSearch Dashboards main menu, select **Discover**.
+1. From the <name> window, select the **<name>** sample dataset. You will see a pre-populated dashboard with line chart visualizations.
+1. Select the ellipsis icon from the **<name>** visualization pane, and then from the **Options** pop-up window, choose **Anomaly Detection** > **Add anomaly detector**.
+1.  Select **Create new detector**.
+1. From the Add anomaly detector window, define the metrics and set thresholds under **Detector details** and **Model features**. To view the visualization within this window, toggle the **Show visualization** button.
+1. _Optional_: Under **Detector details**, select <option> from the dropdown menu to configure the detector details.
+1. _Optional_: Under **Model features**, select <option> from the dropdown menu to configure the evaluation metrics. You can add and delete model features as best suited your use case and data, but you are limited to five model features for that detector.
```

Comment:
I think this can be simplified to just "configure the model features". Prefer to avoid metrics wording since that won't be present in the UI for this flow.